README.md

```diff
@@ -1 +1,52 @@
-# Capstone-project-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project
+
+> This is a websit built by myself to clients who wants to hold a summit and needs a website to make it accessible to users who are interested in forex trading
+
+## Built With
+
+- Html/Css
+- Javascript
+
+## Getting Started
+
+**My project works on mobile and disktop using @Media queries**
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+To get access to this repository, move to "capstone-project" repository, and then either use [ssh] or [https] to
+clone onto your local machine.
+To clone checkout this documentation on github (https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+
+### Live Demo
+
+[My-Capstone-project](https://piesieomane.github.io/Capstone-project-1/)
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author1** with your full name, so as to make your ReadMe more professional